src/harness/unittests/tsserverProjectSystem.ts

```diff
@@ -254,33 +254,34 @@ namespace ts.projectSystem {
     }
 
     export function isFolder(s: FSEntry): s is Folder {
-        return isArray((<Folder>s).entries);
+        return s && isArray((<Folder>s).entries);

```

Comment:
Modified the parameter type if isFile and isFolder to be FSEntry | undefined as the updated checks are right thing to do